screeps-game-api/src/game.rs

```diff
@@ -186,17 +188,80 @@ pub mod map {
     }
 
     /// Implements `Game.map.findExit`.
-    ///
-    /// Does not yet support callbacks.
-    pub fn find_exit(from_room: Room, to_room: Room) -> Result<Exit, ReturnCode> {
-        let code: i32 = js_unwrap!{Game.map.findExit(@{from_room.name()}, @{to_room.name()})};
+    pub fn find_exit(from_room: &Room, to_room: &Room) -> Result<Exit, ReturnCode> {
```

Comment:
Just realized this - sorry for not thinking of this when `find_exit` was originally added: we should allow accepting room names here as well as or instead of `Room` references so we can find exits to rooms not visible.

This is more important for `findRoute` which is much more often used to find routes to far away rooms than it is for `findExit`. Both should be able to accept room names, though, since `Room` objects won't exist for rooms not visible.

My best idea for solving this would be to just accept `&str`, but if you want we could also add a `HasRoomName` trait or something similar implemented both for `Room` and `String` / `str`.